src/multiplex.c

```diff
@@ -79,7 +95,7 @@ void *process_in(struct thread_args *args) {
             switch (read_len) {
                 case 0:
                     fprintf(stderr, "EOF\n");
-                    exit(EXIT_SUCCESS);
+                    exit(EXIT_FAILURE);
```

Comment:
Why? client -> daemon EOF should be normal, I think something like `qubes-gpg-client -k | head` may produce such result (exit early, before retrieving all the data).